Gulpfile.js

```diff
@@ -1,7 +1,6 @@
-var path = require('path');
-var gulp = require('gulp');
-var plugins = require('gulp-load-plugins')();
-var merge = require('merge-stream');
+const gulp = require('gulp');
+const plugins = require('gulp-load-plugins')();
+const merge = require('merge-stream');
```

Comment:
introducing ES6 keywords means project probably won't run on `node < 6`. 
let's add that information in `package.json` under `engines` property.

Reply:
good idea 👍🏽